src/ioc/index.ts

```diff
@@ -1,12 +1,28 @@
-import IoCContainer from 'ioc-lite';
+import IoCContainer from "ioc-lite";
 
-// import { Logger } from '../services/logger';
-// import { HTTP } from '../services/http';
-// import { Users } from '../services/users';
+import { Logger } from "../services/logger";
+import { HTTP } from "../services/http";
+import { Users } from "../services/users";
+import { ApiConfig } from "src/types";
 
-export const createIoCContainer = () =>  {
-  const ioc = new IoCContainer();
-  // you can register some resources right now below...
+export type ServicesType = {
+  logger: typeof Logger;
+  http: typeof HTTP;
+  users: typeof Users;
+  apiConfig: ApiConfig;
+};
+
+export const createIoCContainer = () => {
+  const config: ApiConfig = (window as any).__CONFIG__;
+  delete (window as any).__CONFIG__;
```

Comment:
you shouldn't move this code from `src/index.ts`. We can register resources anywhere we have access to the ioc container.

Reply:
Thank you for your review and comment, the changes have been made